ur5_example.R

```diff
@@ -0,0 +1,13 @@
+#read csv data, add headers

```

Comment:
Address @johmathe 's comments on PR #1 : for example spaces after #
and please follow https://google.github.io/styleguide/Rguide.xml